### DISCUSSION TITLE
Upgrade Standalone .war from Edge Release and Retain Configuration

### DISCUSSION THREAD

**Discussion by judmarc** [kagemomiji/airsonic-advanced#327] [Q&A]
Hi, just migrated from the other airsonic-advanced fork and very happy to see the ongoing work here. I'm using the standalone install, and initially tried simply dropping in the latest .war as a replacement for the one from the other fork. Not being very knowledgeable about all this, in order to avoid errors (service died with a warning about Java 11 when actually Java 17 was installed) I blew away /var/airsonic and reinstalled from scratch. Since there are reasonably frequent updates of the edge releases (which I like), I had two questions:

- Until the next update to the required JDK, should I be able to just substitute the new .war without errors?
- When the next JDK requirement update does occur, is there something short of a new clean install I can do to retain my configuration but avoid errors?

Thanks for the help.

**Comment by hs6666** [comment]
i too would be intrested as I am in the same situation

thanks
